dfetch/commands/check.py

```diff
@@ -48,14 +48,26 @@ def create_menu(subparsers: "argparse._SubParsersAction") -> None:
             action="store_true",
             help="Don't recursively check for child manifests.",
         )
+        parser.add_argument(
+            "projects",
+            metavar="<project>",
+            type=str,
+            nargs="*",
+            help="Specific project(s) to check",
+        )
 
     def __call__(self, args: argparse.Namespace) -> None:
         """Perform the check."""
         manifest, path = dfetch.manifest.manifest.get_manifest()
 
         with in_directory(os.path.dirname(path)):
             exceptions: List[str] = []
-            for project in manifest.projects:
+            projects = manifest.selected_projects(args.projects)
+            if not projects:
```

Comment:
As I understand, the name of the requested project is shown when they aren't found. Maybe, you can also add a list of the existing projects from the manifest. So the user can easily enter a correct project without the need to open the manifest itself.

I presume that you thought about it already, but the generation of this RunTimeError will be repeated several times. So it seems a good candidate for turning generic.

Reply:
Good points!

Reply:
Moved it into manifest class and also added it to the list command